ops_boba/monitor/services/utilities/optimismEnv.js

```diff
@@ -8,14 +8,14 @@ const {
   getContractFactory,
 } = require('@eth-optimism/contracts')
 const fetch = require('node-fetch')
-const { Watcher } = require('../../../../packages/core-utils/dist/watcher')
-
-const addressManagerJSON = require('@eth-optimism/contracts/artifacts/contracts/libraries/resolver/Lib_AddressManager.sol/Lib_AddressManager.json')
```

Comment:
Please use the standard locations (@eth-optimism and @boba/contracts) so we don't have to worry about stale jsons in various locations within the repo